include/nuttx/segger/sysview.h

```diff
@@ -25,18 +25,23 @@
  * Included Files
  ****************************************************************************/
 
-#include <nuttx/config.h>
-#include <nuttx/sched.h>
+#include <nuttx/note/note_driver.h>
 
 /****************************************************************************
- * Pre-processor Definitions
+ * Public Types
  ****************************************************************************/
 
-#ifdef CONFIG_SEGGER_SYSVIEW_PREFIX
-#  define PREFIX(fun) sysview ## _ ## fun
-#else
-#  define PREFIX(fun) fun
-#endif
+struct note_sysview_driver_s
```

Comment:
let's move to the source file and register to the global array in sysview_initialize

Reply:
done